src/deps.deno.ts

```diff
@@ -8,12 +8,24 @@ export type { File } from "https://lib.deno.dev/x/grammy@v1/types.ts";
 
 // Determine whether a file path is absolute
 export { isAbsolute as isAbsolutePath } from "https://deno.land/std@0.135.0/path/mod.ts";
+// Perform a request and returns the raw body
+async function fetchResponseBody(url: string) {
+    const { body } = await fetch(url);
+    if (body === null) throw new Error("Download failed, no response body!");
```

Comment:
v2 could reuse [`InputError`](https://github.com/grammyjs/grammY/blob/32b88e25027c33d7778c0a5c20ecc49f963e8a4c/src/core/input_file.ts#L124) here, should I rename it to `ResponseError`?

Reply:
Sure